src/main/java/Duke.java

```diff
@@ -1,10 +1,88 @@
+import java.util.*;
 public class Duke {
+    private Scanner sc;
+
+    public Duke(Scanner sc) {
+        this.sc = sc;
+    }
+    public void start(Tasklist list) throws DukeException {
+        while(sc.hasNext()) {
```

Comment:
```suggestion
        while (sc.hasNext()) {
```

(while should be followed by a white space)